### PR TITLE
Refactored #content .section.tertiary, #content .section.quaternary

### PR DESCRIPTION
**Pull request by ghost**
Closes department-of-veterans-affairs/vets.gov-team#787

- Moved #content .section.coda to l-playbook and simplified selector to .coda
- Moved #content .section.tertiary, #content .section.quaternary to l-playbook. Simplified selectors.
- Moved .splash--alternate block to l-playbook.
- Removed unused .splash--app block, .splash--blog blocks.
- Simplified #playbook .coda [href^=http] selector.
- Consolidated #content .section.secondary blocks.

Replaced $primary-color, $secondary-color variables

Reduced number of color variables, variable names.
Closes department-of-veterans-affairs/vets.gov-team#748

- Replaced $primary-color with $color-primary
- Replaced $secondary-color with $color-gray-lighter
- Removed unused selector blocks containing $secondary-color
   - Removed .read-more CSS block.
   - Removed .methodology .medium-3.hcd h4 block. Unused.
   - Removed m-checklist.scss partial. Unused.

### CODE REVIEW

**Comment by jcmeloni-usds** [comment]
@tiffanybbrown-va I'm ok merging this unless you want another LGTM from @melwoodard or someone. just LMK and I'll press the appropriate button.

**Review by robbiethegeek-usds — APPROVED** [review]

**Comment by robbiethegeek-usds** [comment]
Once the tests pass please feel free to merge @tiffanybbrown-va

**Pull request merged**
